src/checks/badpackets/regen/RegenB.php

```diff
@@ -53,15 +53,15 @@ public function maxViolations() : int {
 
 	public function checkEvent(Event $event, PlayerAPI $playerAPI) : void {
 		if ($event instanceof EntityRegainHealthEvent) {
-			if ($event->getRegainReason() != EntityDamageEvent::CAUSE_MAGIC && $event->getRegainReason() != EntityDamageEvent::CAUSE_CUSTOM) {
+			if (!in_array($event->getRegainReason(), [EntityDamageEvent::CAUSE_MAGIC, EntityDamageEvent::CAUSE_CUSTOM], true)) {
 				$tick = (double) Server::getInstance()->getTick();
-				$tps = (double) Server::getInstance()->getTicksPerSecond();
```

Comment:
Why `(double)` is removed?

Reply:
because 
```php
/**
 * Returns the last server TPS measure
 */
public function getTicksPerSecond() : float{
	return round($this->currentTPS, 2);
}
```

double cast is useless